_people/sophy_baxter.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Sophy Baxter
+submitted: 2020-03-02 
+gender: female 
+# headshot: 
+course: BSc Nursing 
+graduated: 2022
```

Comment:
did we fix the fact that putting in a grad year in the future would automatically label them as graduated?

Reply:
Yes I think so, thanks to @emmambarber 😛